src/app/services/images.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { AngularFireStorage, AngularFireUploadTask } from '@angular/fire/storage';
+import { UploadTask } from '@angular/fire/storage/interfaces';
+
+@Injectable()
+export class ImagesService {
+  uri = '/cards';
+
+  constructor(private firebase: AngularFireStorage) {
+  }
+
+  upload(file: any): UploadTask {
+    const ref = this.firebase.ref(file.name);
+    return this.firebase.storage.ref(file.name).put(file);
+  }
```

Comment:
Does this throw an error? Check if error response handling is needed.

Reply:
It doesn't seem to result in error from documentation.